src/Umbraco.Web/PropertyEditors/ValueConverters/MultiUrlPickerValueConverter.cs

```diff
@@ -32,7 +32,9 @@ public MultiUrlPickerValueConverter(IPublishedSnapshotAccessor publishedSnapshot
 
         public override PropertyCacheLevel GetPropertyCacheLevel(IPublishedPropertyType propertyType) => PropertyCacheLevel.Snapshot;
 
-        public override bool? IsValue(object value, PropertyValueLevel level) => value?.ToString() != "[]";
+        public override bool? IsValue(object value, PropertyValueLevel level) => value is not null &&
+            !string.IsNullOrWhiteSpace(value.ToString()) &&
+            value.ToString() != "[]";
```

Comment:
```suggestion
        public override bool? IsValue(object value, PropertyValueLevel level) => value?.ToString() is { Length: > 0 } valueStr && valueStr != "[]";
```
Wow, nice to see that the project has recent C# features enabled! 😯
Is there a need to actually check for whitespaces? Otherwise we can just check if it's an empty string.